src/linker/Linker/RemoveAttributeInstancesAttribute.cs

```diff
@@ -14,14 +16,14 @@ namespace Mono.Linker
 		AttributeTargets.Class, Inherited = false)]
 	public sealed class RemoveAttributeInstancesAttribute : Attribute
 	{
-		public RemoveAttributeInstancesAttribute ()
+		public RemoveAttributeInstancesAttribute (System.Collections.Generic.ICollection<CustomAttributeArgument> values)
```

Comment:
Why ICollection instead of `Collection<CustomAttributeArgument>` ?

Reply:
No particular reason, would it be better to use `Collection<CustomAttributeArgument>`?

Reply:
Yes, less interface dispatch